[2020-06-17 15:40:24] INFO [<module>: config.py, 10] hello
[2020-06-17 15:50:08] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-17 15:50:08] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 21
[2020-06-17 15:51:08] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-17 15:51:08] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 21
[2020-06-17 15:52:22] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-17 15:52:22] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 21
[2020-06-17 15:52:22] INFO [test_user_login_normal: test_user_login.py, 22] 测试用例：test_user_login_normal
[2020-06-17 15:52:22] INFO [test_user_login_normal: test_user_login.py, 23] url：http://115.28.108.130:5000/api/user/login/
[2020-06-17 15:52:22] INFO [test_user_login_normal: test_user_login.py, 24] 请求参数：{"name": "张三","password":"123456"}
[2020-06-17 15:52:22] INFO [test_user_login_normal: test_user_login.py, 25] 期望结果：<h1>登录成功</h1>
[2020-06-17 15:52:22] INFO [test_user_login_normal: test_user_login.py, 26] 实际结果：<h1>登录成功</h1>
[2020-06-17 15:58:25] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-17 15:58:25] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 21
[2020-06-17 15:58:25] INFO [log_case_info: case_log.py, 7] 测试用例：test_user_login_normal
[2020-06-17 15:58:25] INFO [log_case_info: case_log.py, 8] url：http://115.28.108.130:5000/api/user/login/
[2020-06-17 15:58:25] INFO [log_case_info: case_log.py, 9] 请求参数：{"name": "张三","password":"123456"}
[2020-06-17 15:58:25] INFO [log_case_info: case_log.py, 10] 期望结果：<h1>登录成功</h1>
[2020-06-17 15:58:25] INFO [log_case_info: case_log.py, 11] 实际结果：<h1>登录成功</h1>
[2020-06-17 17:39:59] INFO [<module>: run_all.py, 6] ================================== 测试开始 ==================================
[2020-06-17 17:40:00] INFO [send_email: send_email_m.py, 25] 邮件发送完成！
[2020-06-17 17:40:00] INFO [<module>: run_all.py, 13] ================================== 测试结束 ==================================
[2020-06-18 10:42:38] INFO [<module>: run_all.py, 6] ================================== 测试开始 ==================================
[2020-06-18 10:42:38] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-18 10:42:38] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 21
[2020-06-18 10:42:38] INFO [log_case_info: case_log.py, 9] 测试用例：test_user_login_normal
[2020-06-18 10:42:38] INFO [log_case_info: case_log.py, 10] url：http://115.28.108.130:5000/api/user/login/
[2020-06-18 10:42:38] INFO [log_case_info: case_log.py, 11] 请求参数：{"name": "张三","password":"123456"}
[2020-06-18 10:42:38] INFO [log_case_info: case_log.py, 12] 期望结果：<h1>登录成功</h1>
[2020-06-18 10:42:38] INFO [log_case_info: case_log.py, 13] 实际结果：<h1>登录成功</h1>
[2020-06-18 10:42:38] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-18 10:42:38] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 27
[2020-06-18 10:42:44] INFO [send_email: send_email_m.py, 24] 邮件发送完成！
[2020-06-18 10:42:44] INFO [<module>: run_all.py, 13] ================================== 测试结束 ==================================
[2020-06-18 10:43:54] INFO [<module>: run_all.py, 6] ================================== 测试开始 ==================================
[2020-06-18 10:43:54] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-18 10:43:55] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 21
[2020-06-18 10:43:55] INFO [log_case_info: case_log.py, 9] 测试用例：test_user_login_normal
[2020-06-18 10:43:55] INFO [log_case_info: case_log.py, 10] url：http://115.28.108.130:5000/api/user/login/
[2020-06-18 10:43:55] INFO [log_case_info: case_log.py, 11] 请求参数：{"name": "张三","password":"123456"}
[2020-06-18 10:43:55] INFO [log_case_info: case_log.py, 12] 期望结果：<h1>登录成功</h1>
[2020-06-18 10:43:55] INFO [log_case_info: case_log.py, 13] 实际结果：<h1>登录成功</h1>
[2020-06-18 10:43:55] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-18 10:43:55] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 27
[2020-06-18 10:43:56] INFO [send_email: send_email_m.py, 24] 邮件发送完成！
[2020-06-18 10:43:56] INFO [<module>: run_all.py, 13] ================================== 测试结束 ==================================
[2020-06-18 10:45:10] INFO [<module>: run_all.py, 6] ================================== 测试开始 ==================================
[2020-06-18 10:45:10] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-18 10:45:10] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 21
[2020-06-18 10:45:10] INFO [log_case_info: case_log.py, 9] 测试用例：test_user_login_normal
[2020-06-18 10:45:10] INFO [log_case_info: case_log.py, 10] url：http://115.28.108.130:5000/api/user/login/
[2020-06-18 10:45:10] INFO [log_case_info: case_log.py, 11] 请求参数：{"name": "张三","password":"123456"}
[2020-06-18 10:45:10] INFO [log_case_info: case_log.py, 12] 期望结果：<h1>登录成功</h1>
[2020-06-18 10:45:10] INFO [log_case_info: case_log.py, 13] 实际结果：<h1>登录成功</h1>
[2020-06-18 10:45:10] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 115.28.108.130:5000
[2020-06-18 10:45:10] DEBUG [_make_request: connectionpool.py, 433] http://115.28.108.130:5000 "POST /api/user/login/ HTTP/1.1" 200 27
[2020-06-18 10:45:15] INFO [send_email: send_email_m.py, 24] 邮件发送完成！
[2020-06-18 10:45:15] INFO [<module>: run_all.py, 13] ================================== 测试结束 ==================================
